frog/imports/client/Wiki/index.js

```diff
@@ -52,13 +52,19 @@ export const LearningItem = dataFn.LearningItem;
 
 type WikiCompPropsT = {
   location: *,
-  match: {
+  match?: {
     params: {
       wikiId: string,
-      pageTitle: ?string
+      pageTitle: ?string,
+      instance?: string
     }
   },
-  history: Object
+  history: Object,
+  embedPage?: {
```

Comment:
What is the difference between embedPage and match? Could we merge the two if they contain the same props?

Reply:
We would still need a prop to signify whether the Wiki is embedded or not. Also `match` is kind of a special prop passed by the router so I wouldn't recommend to override that.